receiver.go

```diff
@@ -58,8 +60,7 @@ func (r *receiver) receiveLoop() error {
 			if err == io.EOF {
 				r.context.logFrame("receiveLoop stopped")
 			} else if r.parseError != nil {
-				r.context.log("Error: receiveLoop exiting due to parse error: %v", r.parseError)
-				err = fmt.Errorf("BLIP frame parse error: %v", r.parseError)
+				err = r.parseError
 			} else {
 				r.context.log("Error: receiveLoop exiting with WebSocket error: %v", err)
```

Comment:
I noticed a case where the error returned from `receiveLoop()` was being ignored.  I am pushing up a commit to this PR branch to fix.